Include/pythonrun.h

```diff
@@ -64,25 +64,7 @@ PyAPI_FUNC(struct _mod *) PyParser_ASTFromFileObject(
     PyCompilerFlags *flags,
     int *errcode,
     PyArena *arena);
-#endif
 
-#ifndef PyParser_SimpleParseString
```

Comment:
These declarations need to be restored as well

Reply:
Done.